drbd/defaults.yaml

```diff
@@ -20,6 +20,9 @@ drbd:
       quorum: "off"
     net:
       multi_primaries: ""
+      max_buffers: ""
```

Comment:
These defaults looks strange. They shouldn't actually be default values? I mean, what is the value you set if nothing is defined here?
I see some predefined values now in `global_confs.sls`, shouldn't we put those?

Reply:
The `defaults` in global_confs.sls is the default values defined in current DRBD code if configured.
The values in `defaults.yaml` will overlap the `defaults` one actually, since more of them are not necessary to be configured. (1. In case default values changed but not updated here.  2. Some parameters should not be configured in some cases, even with default value. Like `fencing` only should work with pacemaker.)

That is the reason i kept defaults.yaml

Reply:
After discussion, i agree to remove them in `defaults.yaml` sounds more reasonable than using empty strings.